kiva/tests/test_qpainter_drawing.py

```diff
@@ -42,11 +42,6 @@ def create_graphics_context(self, width, height):
 
         return GraphicsContext((width, height))
 
-    @unittest.expectedFailure
-    def test_image(self):
-        """ QPainter interprets images as BGRA. """
-        super().test_image()
```

Comment:
i'm not entirely sure why this test was removed - instead of unmarking it as an expected failure.

Reply:
It only existed here so that it could be marked with `expectedFailure`. It's no longer needed here.